saleor/discount/models.py

```diff
@@ -50,8 +50,8 @@ class Voucher(models.Model):
     code = models.CharField(max_length=12, unique=True, db_index=True)
     usage_limit = models.PositiveIntegerField(null=True, blank=True)
     used = models.PositiveIntegerField(default=0, editable=False)
-    start_date = models.DateField(default=date.today)
-    end_date = models.DateField(null=True, blank=True)
+    start_date = models.DateTimeField(default=timezone.now)
```

Comment:
You should consider a different name. `start_date` was good when we operate on DateField. It can be misleading now

Reply:
I don't think it's a big issue; from what I see some other APIs also use "date" fields for representing DateTime. We also have this case e.g. for `User.date_joined` field and I don't recall there were ever problems with that.